### PR TITLE
Adds Twitter/github/facebook/linkedin profiles and uses sidekiq instead of delayedjob

### PR DESCRIPTION
**Pull request by heroic**
Refs #9, #13

### CODE REVIEW

**Comment by unamashana** [comment]
@heroic in general it's better to create separate issues for different tasks and then merge them separately. Mixing functionality makes it harder to test. Thoughts @avinasha ?

File: Gemfile [hunk 1]
```diff
@@ -1,8 +1,9 @@
 source 'http://rubygems.org'
 
-ruby '1.9.3'
+ruby '2.0.0'
```
Comment (unamashana) [inline Gemfile:3]: too many moving parts in this commit

**Comment by avinasha** [comment]
@prateekdayal makes sense... @heroic can you split this commit into diff issues? It will help us merge and deploy.. Thanks

**Comment by heroic** [comment]
Sure

**Pull request closed without merging**